Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private readonly string _name;
+        private readonly decimal[] _balance;
```

Comment:
Yes, that's exactly what we were expecting :)

Reply:
Thank you!